lib/services.js

```diff
@@ -252,8 +225,11 @@ export default {
                     log.warn('error parsing acl', { error: err });
                     return cb(err);
                 }
-                omVal.acl = parsedACL;
-                metadata.putObjectMD(bucketName, objectKey, omVal, log, err => {
+                md.setAcl(parsedACL);
+                log.trace('object metadata', {
+                    omVal: md.getData(),
```

Comment:
you're right :) i have replace what exists before, but we can consider to remove this :) Here its a trace log, so it will be log only if there is an error, no opinion on this one :)